src/main/java/sg/ncl/MainController.java

```diff
@@ -461,11 +461,16 @@ public String loginSubmit(
             String userStatus = user.getStatus();
             boolean emailVerified = user.getEmailVerified();
             if (!emailVerified || (UserStatus.CREATED.toString()).equals(userStatus)) {
+                redirectAttributes.addAttribute("statuschecklist", userStatus);
                 log.info("User {} not validated, redirected to email verification page", id);
-                return "redirect:/email_not_validated";
+                return "redirect:/email_checklist";
             } else if ((UserStatus.PENDING.toString()).equals(userStatus)) {
+                redirectAttributes.addAttribute("statuschecklist", userStatus);
                 log.info("User {} not approved, redirected to application pending page", id);
-                return "redirect:/team_application_under_review";
+                return "redirect:/email_checklist";
+//            } else if ((UserStatus.PENDING.toString()).equals(userStatus)) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3ACommentedOutCodeLine)